aws/resource_aws_cloudwatch_event_target_test.go

```diff
@@ -123,6 +123,12 @@ func TestAccAWSCloudWatchEventTarget_basic(t *testing.T) {
 						regexp.MustCompile(fmt.Sprintf(":%s$", snsTopicName2))),
 				),
 			},
+			{
+				ResourceName:      "aws_cloudwatch_event_target.moobar",
+				ImportState:       true,
+				ImportStateIdFunc: testAccAWSCloudWatchEventTargetImportStateIdFunc("aws_cloudwatch_event_target.moobar"),
+				ImportStateVerify: true,
+			},
```

Comment:
Hi @chaspy - thanks so much for this contribution! Overall, this looks great, but for the sake of completeness, could you add this block to all of the `TestAccAWSCloudWatchEventTarget` acceptance tests in this file?

```
{
				ResourceName:      "aws_cloudwatch_event_target.moobar",
				ImportState:       true,
				ImportStateIdFunc: testAccAWSCloudWatchEventTargetImportStateIdFunc("aws_cloudwatch_event_target.moobar"),
				ImportStateVerify: true,
			},
```

You'll want to make sure that `ResourceName` matches for each test since it looks like they vary a bit from test to test. If you're able to add these and re-run the acceptance tests for us, we can go ahead and get this merged in. Thanks! 👍